### PR TITLE
Fix package name for ConsoleContextLayout doc & bug in CCL

### PR DESCRIPTION
**Pull request by FunFunFine**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ import com.typesafe.sbt.SbtGit.git
 
 moduleName := "tofu"
 
-val libVersion = "0.9.1"
+val libVersion = "0.9.3"
```
Comment (catostrophe) [inline build.sbt:6]: Why?

File: logging/layout/src/main/scala/tofu/logging/logback/ConsoleContextLayout.scala [hunk 2]
```diff
@@ -31,7 +31,11 @@ class WrappedEvent(event: ILoggingEvent) extends ILoggingEvent {
       case ContextMarker(ctx, _) => intoMdc(ctx)
       case _                     =>
     }
-    for (arg <- event.getArgumentArray) intoMdc(arg)
+    val argumentArray: Array[AnyRef] = event.getArgumentArray
```
Comment (catostrophe) [inline logging/layout/src/main/scala/tofu/logging/logback/ConsoleContextLayout.scala:34]: Simply
```scala
for (arr <- Option(event.getArgumentArray); arg <- arr) intoMdc(arg)
```

File: build.sbt (hunk 1 above)
Comment (FunFunFine) [reply build.sbt:6]: Because it is a bugfix.
Semver tells to update minor version on bugfixes AFAIR

Comment (catostrophe) [reply build.sbt:6]: It's not you (or another contributor) who bumps the version. Please remove it and leave to those who prepare the release

Comment (FunFunFine) [reply build.sbt:6]: Got it.
```suggestion
val libVersion = "0.9.1"
```

**Review by catostrophe — DISMISSED** [review]

**Review by catostrophe — APPROVED** [review]

**Pull request merged**